jujugui/static/gui/src/app/components/entity-details/test-entity-details.js

```diff
@@ -115,7 +78,7 @@ describe('EntityDetails', function() {
                   'getEntity function not called');
     assert.equal(getEntity.args[0][0], id,
                  'getEntity not called with the entity ID');
-    assert.deepEqual(output,
+    var expected = (
```

Comment:
Why this pattern? Seems strange to assign to a variable that only gets used once. We don't use this pattern elsewhere.